app/src/keymap.c

```diff
@@ -179,17 +179,23 @@ int zmk_keymap_apply_position_state(uint8_t source, int layer, uint32_t position
         .timestamp = timestamp,
     };
 
-    LOG_DBG("layer: %d position: %d, binding name: %s", layer, position,
-            log_strdup(binding.behavior_dev));
+    return zmk_run_behavior(&binding,event,source,pressed);
+}
+
+int zmk_run_behavior(struct zmk_behavior_binding *binding, struct zmk_behavior_binding_event event,uint8_t source,bool pressed){
```

Comment:
`binding` probably should be `const` here. Also, be sure to run `clang-format` on any changed files for consistent formatting.